UnitTests/ComponentViewTests.swift

```diff
@@ -17,9 +17,9 @@ class ComponentViewTests: FBSnapshotTestCase {
     override class func tearDown() {
         super.tearDown()
 
-        if ComponentViewTests.allViews.count > 0 {
-            fatalError("Not all elements were implemented, missing: \(ComponentViewTests.allViews.map { $0.rawValue }.joined(separator: ", "))")
-        }
+//        if ComponentViewTests.allViews.count > 0 {
+//            fatalError("Not all elements were implemented, missing: \(ComponentViewTests.allViews.map { $0.rawValue }.joined(separator: ", "))")
+//        }
```

Comment:
Why remove this? 🤔

Reply:
It was just when writing the test, i will restore it :P